src/AWS.Deploy.ServerMode.Client/Exceptions/InternalServerModeException.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.\r
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+
+namespace AWS.Deploy.ServerMode.Client.Exceptions
+{
+    /// <summary>
+    /// Exception is thrown when deployment tool server failed to start for un unknown reason.
+    /// </summary>
+    public class InternalServerModeException : Exception
+    {
+        public InternalServerModeException(string message): base(message)
```

Comment:
Can you use the same overloads like all our other exceptions? (message and inner exception)
Also why are we creating a new .cs file for every new exception?

Reply:
We don't have inner exception there we don't need inner exception overload here.

I moved exceptions to Exceptions.cs like we to in CLI.